docs/integrations/destinations/bigquery.md

```diff
@@ -133,10 +133,11 @@ Now that you have set up the BigQuery destination connector, check out the follo
 
 | Version | Date       | Pull Request                                               | Subject                                                                                         |
 |:--------|:-----------|:-----------------------------------------------------------|:------------------------------------------------------------------------------------------------|
-| 1.1.12 | 2022-00-02 | [\#15180](https://github.com/airbytehq/airbyte/pull/15180) | Fix standard loading mode  |
-| 1.1.11 | 2022-06-24 | [\#14114](https://github.com/airbytehq/airbyte/pull/14114) | Remove "additionalProperties": false from specs for connectors with staging  |
-| 1.1.10 | 2022-06-16 | [\#13852](https://github.com/airbytehq/airbyte/pull/13852) | Updated stacktrace format for any trace message errors |
-| 1.1.9  | 2022-06-17 | [\#13753](https://github.com/airbytehq/airbyte/pull/13753) | Deprecate and remove PART_SIZE_MB fields from connectors based on StreamTransferManager  |
+| 1.1.13  | 2022-00-02 | [15180](https://github.com/airbytehq/airbyte/pull/15180) | Fix standard loading mode  |
+| 1.1.12  | 2022-08-02 | [14801](https://github.com/airbytehq/airbyte/pull/14801) | Fix multiply log bindings |
```

Comment:
This new version for this PR should be `1.1.13`. `1.1.12` is already allocated to `Fix standard loading mode`. This is fixed in https://github.com/airbytehq/airbyte/pull/15409.